generators/client/templates/src/main/webapp/app/services/auth/_auth.jwt.service.js

```diff
@@ -29,13 +29,45 @@
         }
 
         function login (credentials) {
-            var data = {
+            <% if(authenticationType === 'uaa') { %>
+              var data = {
+                username: credentials.username,
+                password: credentials.password,
+                grant_type: "password"
+              };
+              var headers = {
+                'Content-Type': 'application/x-www-form-urlencoded',
+                "Authorization" : "Basic d2ViX2FwcDo="
```

Comment:
Is this the base 64-encoded "client_id:client_secret"? Shouldn't be hardcoded, should it?